opencti-platform/opencti-graphql/package.json

```
@@ -20,6 +20,8 @@
     "serv": "node build/back.js",
     "insert:dev": "node build/script-insert-dataset.js",
     "test:dev": "vitest watch --config vitest.config.dev.ts",
```

Now that I'm playing a lot with tests, I hesitate to have both watch and not watch, something like:
```suggestion
     "test:watch": "vitest watch --config vitest.config.dev.ts",
    "test:dev": "vitest run --config vitest.config.dev.ts",
```

---

yes, makes sense. I'm rarely using watch mode personally.